### PR TITLE
Added Copyright template - COPYRIGHT.md

### PR DESCRIPTION
**Pull request by michalbundyra**
As it was agreed a long time ago .... we need COPYRIGHT.md file and then in headers we are going to have reference to that file.

As I'm updating zend-coding-standard we need template in one common place.

### CODE REVIEW

File: COPYRIGHT.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Copyright (c) 2015-2018, Zend Technologies USA, Inc.
```
Comment (weierophinney) [inline COPYRIGHT.md]: Needs the following line after it:

```text
All rights reserved.
```

The start year on this one should probably be 2005, as it references the project, and not necessarily the repository.

File: LICENSE.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2017, Zend Technologies USA, Inc.
+Copyright (c) 2015-2018, Zend Technologies USA, Inc.
 All rights reserved.
```
Comment (weierophinney) [inline LICENSE.md]: Sucks that we have to repeat it here, but that's how the BSD license is phrased. So we'll have it in two locations, essentially.

File: bin/zf-maintainer [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <?php
 /**
  * @see       https://github.com/zendframework/maintainers for the canonical source repository
- * @copyright Copyright (c) 2017-2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright https://github.com/zendframework/maintainers/blob/master/COPYRIGHT.md Copyright
```
Comment (weierophinney) [inline bin/zf-maintainer]: Here and all others referencing the repo's copyright file: we do not need the word "Copyright" at the end; it's already in the annotation name.

File: template/COPYRIGHT.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Copyright (c) {year}, Zend Technologies USA, Inc.
```
Comment (weierophinney) [inline template/COPYRIGHT.md]: Needs the following line after it:

```text
All rights reserved.
```

**Review by weierophinney — CHANGES_REQUESTED** [review]

File: COPYRIGHT.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Copyright (c) 2005-2018, Zend Technologies USA, Inc. All rights reserved.
```
Comment (webimpress) [inline COPYRIGHT.md]: @weierophinney Do we need to add here also link to www.zend.com?

Current copyright tag description is:
`Copyright (c) 2005-2017 Zend Technologies USA Inc. (https://www.zend.com)`.

Comment (webimpress) [reply COPYRIGHT.md]: @weierophinney It's updated now.

**Pull request closed without merging**